### PR TITLE
Revert "Revert "Cvsl 1327 dps header update for cvl""

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**
Reverts ministryofjustice/create-and-vary-a-licence#677

### CODE REVIEW

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: server/config.ts [hunk 1]
```diff
@@ -164,6 +164,15 @@ export default {
        */
       licencesUrl: get('LICENCES_URL', 'http://host.docker.internal:3000', requiredInProduction),
     },
+    frontendComponents: {
+      url: get('COMPONENT_API_URL', 'http://localhost:3000', requiredInProduction),
+      timeout: {
+        response: Number(get('COMPONENT_API_TIMEOUT_SECONDS', 10000)),
+        deadline: Number(get('COMPONENT_API_TIMEOUT_SECONDS', 10000)),
+      },
```
Comment (andrewrlee) [inline server/config.ts:172]: can we change these default timeouts to 2000? 

Obviously this value was suggested, but long timeouts are a bit useless here. waiting 10 seconds to show the header is akin to an outage

Comment (nishanthmahasamudram) [reply server/config.ts:172]: Yes, you are right.
I will changes the timeout to 2 seconds.

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/views/layout.njk [hunk 2]
```diff
@@ -40,10 +40,22 @@
     <!-- End Google Tag Manager -->
   {% endif %}
 
+  {% if feComponents.jsInclude %}
```
Comment (whitfield-mj) [inline server/views/layout.njk:43]:
```suggestion
  {% if feComponents.jsIncludes %}
```

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**